src/Swup/Resources/assets/src/controller.js

```diff
@@ -23,7 +23,9 @@ export default class extends Controller {
             cache: this.element.hasAttribute('data-cache'),
             animateHistoryBrowsing: this.element.hasAttribute('data-animate-history-browsing'),
             plugins: [
-                'slide' === this.element.hasAttribute('data-theme') ? new SwupSlideTheme() : new SwupFadeTheme(),
+                this.element.hasAttribute('data-theme') && 'slide' === this.element.getAttribute('data-theme')
```

Comment:
You should be able to just use `'slide' === this.element.getAttribute('data-theme')` as getAttribute will just return null if no attribute exists

Reply:
@tgalopin fixed :)